src/diagnostics/cmake.ts

```diff
@@ -5,9 +5,10 @@
 import { Logger } from '@cmt/logging';
 import { OutputConsumer } from '@cmt/proc';
 import * as util from '@cmt/util';
+import { oneLess } from '@cmt/basic/util';
 import * as vscode from 'vscode';
 
-import { FileDiagnostic, oneLess } from './util';
+import { FileDiagnostic } from './util';
 
 /**
  * Class which consumes output from CMake.
```

Comment:
Should these two lines be modified as well or not? 

Line 96                     `const vsdiag = new vscode.Diagnostic(new vscode.Range(lineno, 0, lineno, 9999), full, diagmap[level]);`

Line 157 `const related = new vscode.DiagnosticRelatedInformation(new vscode.Location(fileUri, new vscode.Range(lineNo, 0, lineNo, 999)), "In call to '${command}' here" );`